### PR TITLE
MGDOBR-211 Publish operator image built with Minikube profile

### PR DESCRIPTION
**Pull request by sutaakar**
Signed-off-by: Karel Suta <ksuta@redhat.com>

[MGDOBR-211](https://issues.redhat.com/browse/MGDOBR-211) 

Untested as the script is triggered by main branch push. Implementation inspired by other commands in file, hopefully should be syntactically correct.
The operator is rebuilt as we need to produce application using different profile, pushed into same repository with tag prefix.

Please make sure that your PR meets the following requirements:

- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config)
- [x] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] All new functionality is tested
- [x] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: .github/workflows/CD.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ jobs:
       MAVEN_OPTS: "-Dhttps.protocols=TLSv1.2 -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=WARN -Dorg.slf4j.simpleLogger.showDateTime=true -Djava.awt.headless=true"
       FLEET_MANAGER_CONTAINER_NAME: quay.io/5733d9e2be6485d52ffa08870cabdee0/fleet-manager:${{ github.sha }}
       FLEET_SHARD_CONTAINER_NAME: quay.io/5733d9e2be6485d52ffa08870cabdee0/fleet-shard:${{ github.sha }}
+      FLEET_SHARD_MINIKUBE_CONTAINER_NAME: quay.io/5733d9e2be6485d52ffa08870cabdee0/fleet-shard:k8s-${{ github.sha }}
```
Comment (r00ta) [inline .github/workflows/CD.yml:22]: Do you think we should set the tag of the ocp image like `ocp-..` ?

**Review by r00ta — APPROVED** [review]

File: .github/workflows/CD.yml (hunk 1 above)
Comment (sutaakar) [reply .github/workflows/CD.yml:22]: Yes, good idea. Will allow simple identification of the image tag.

Comment (sutaakar) [reply .github/workflows/CD.yml]: Done.

**Review by r00ta — APPROVED** [review]
Thank you!

**Pull request merged**